app/controllers/api/v1/reports_controller.rb

```diff
@@ -38,7 +37,7 @@ def destroy
       def last
         conditions = { :host_id => Host.authorized(:view_hosts).friendly.find(params[:host_id]).try(:id) } if params[:host_id].present?
         max_id = resource_scope.where(conditions).maximum(:id)
-        @report = resource_scope.includes(:logs => [:message, :source]).find(max_id)
+        @report = resource_scope.find(max_id)
```

Comment:
You are right, I'm am wrong: didn't realized the `though` relation there. I've seen no performance impacts both ways so reverting this part of patch